.github/workflows/webviz-core-components.yml

```diff
@@ -33,12 +33,16 @@ jobs:
 
       - name: 📦 Install build dependencies
         run: |
+          cd ./react
           npm ci --ignore-scripts
+          npm run copy-package-json
+          cd ..
           pip install .[dependencies]
           pip install dash[dev]
   
       - name: 🏗️ Build JavaScript part
         run: |
+          cd ./react
           npm run build
```

Comment:
```suggestion
          npm run build --prefix ./react
```

Reply:
@anders-kiaer Unfortunately not working here since the `dash-generate-components` command cannot be found.